core/commands/partials/vagrant_cleaner.rb

```diff
@@ -180,38 +151,4 @@ def destroy_virtualbox_machine(configuration, node, vbox_name = nil)
     check_command("VBoxManage unregistervm #{vbox_name} -delete",
                   "Unable to delete #{vbox_name} machine.")
   end
-
-  # Destroy the aws virtual machine.
-  # To destroy the node by name, use aws_box_name param.
-  #
-  # @param configuration [Configuration] configuration to user.
-  # @param node [String] name of node to destroy.
-  # @param aws_box_name [String] name of instance to destroy.
-  def destroy_aws_machine(configuration, node, aws_box_name = nil)
-    aws_box_name ||= "#{configuration.name}_#{node}"
-    instance_id = get_aws_instance_id_by_node_name(aws_box_name)
-    if instance_id.nil?
-      @ui.error("Unable to terminate #{aws_box_name} machine. Instance id does not exist.")
-      return
-    end
-    unless @aws_service.instance_running?(instance_id)
-      @ui.info("AWS instance '#{instance_id}' for node '#{node}' is not running.")
-    end
-    @ui.info("Sending termination command for instance '#{instance_id}' used for node '#{node}.")
-    @aws_service.terminate_instance(instance_id)
```

Comment:
The corresponding method of the AWS client is no longer used. Please remove it too.